src/renderer/components/inputs/TextAreaInput.tsx

```diff
@@ -26,19 +39,54 @@ export const TextAreaInput = memo(
         }, 500);
 
         return (
-            <Textarea
+            <Resizable
                 className="nodrag"
-                disabled={isLocked}
-                draggable={false}
-                minW={240}
-                placeholder={label}
-                resize={resizable ? 'both' : 'none'}
-                value={tempText}
-                onChange={(event) => {
-                    setTempText(event.target.value);
-                    handleChange(event);
+                defaultSize={size}
+                enable={{
+                    top: false,
+                    right: !isLocked && resizable && true,
+                    bottom: !isLocked && resizable && true,
+                    left: false,
+                    topRight: false,
+                    bottomRight: !isLocked && resizable && true,
+                    bottomLeft: false,
+                    topLeft: false,
+                }}
+                grid={useMemoArray<[number, number]>(
+                    isSnapToGrid ? [snapToGridAmount, snapToGridAmount] : [1, 1]
+                )}
```

Comment:
Why should the size of text boxes respect snap to grid?

Reply:
Idk, I did it for iterators so I just also did it here. I figured it made sense at the time that everything would just snap to the grid including resizing

Reply:
That also seems a bit odd to me, but I think it's justifiable there. However, I don't see why inputs should snap to the grid. After all, the inputs don't directly determine the size of the whole node, so the size of the node isn't actually aligned with the grid most of the time.

Reply:
True. I guess I was thinking about it in terms of just the node node since it's the only input